submissions/korotushak96/a-tiny-JS-world/index.js

```diff
@@ -27,32 +27,32 @@
    print('human; <strong>John</strong>; male; 2; 2; <em>Hello world!</em>; Rex, Tom, Jenny');
    print('human; <strong>John</strong>; male; 2; 2; <em>Hello world!</em>; Rex, Tom, Jenny', 'div');
    */
-  const dog = {
+const dog = {
     species: 'dog',
```

Comment:
species: `dog`, `cat` and `human` are repeated. You can define object for constants.

Reply:
Also you can define as constants some other repeated parameters, how about `gender` ?